var/spack/repos/builtin/packages/gearshifft/package.py

```diff
@@ -28,32 +31,43 @@ class Gearshifft(CMakePackage):
             description='use OpenMP parallel fftw libraries')
     # variant('hcfft', default=True,
     #         description='Not implemented yet')
+    variant('mkl', default=True,
+            description='Compile gearshifft_fftwwrappers')
```

Comment:
A couple of questions:
1. Does this package depend on any virtual API that is not yet declared (like `lapack` or `blas`)?
2. Can this package depend on both `fftw` and `intel-mkl` at the same time?
I'm trying to understand how `intel-mkl` is used here to check if we can avoid the variant above.

Reply:
1. It depends on the FFTW wrapper library provided by MKL
2. Yes, each variant is compiled into a separate executable.

Reply:
I mean it uses MKL's FFTW wrapper library in case MKL is used.

Reply:
Ok. Would it make sense then to rework this to have
```python
depends_on('fftw-api')
```
and drop explicit dependency on both `intel-mkl` and `fftw`? Then in the recipe we can:
```python
if spec.satisfies('^intel-mkl'):
    # Do something specific for Intel-mkl
```
Would this model fit better the actual dependencies of the package?

Reply:
Ok, thanks for clarifying that. I missed:

> Yes, each variant is compiled into a separate executable.

Reading from the email.